using_images/xpaas_images/a_mq.adoc

```diff
@@ -5,4 +5,92 @@
 :icons:
 :experimental:
 
-Information on the Red Hat JBoss A-MQ xPaaS image will be added soon.
+Red Hat JBoss A-MQ (JBoss A-MQ) is available as a containerized xPaaS image that is designed for use with OpenShift v3. It allows developers to quickly deploy an A-MQ message broker in a hybrid cloud environment.
```

Comment:
Suggest dropping "v3" here, and/or conditionalize it between "OpenShift Enterprise 3.0" and <something> for Origin.